ideainspect.groovy

```diff
@@ -85,13 +74,20 @@ if (!resultPath.mkdirs()) {
 }
 
 //
+// --- Running VMOptions Modifier
+//
+if(cliOpts.sc){
+  modifyIdeaProps(cliOpts)
```

Comment:
We should ensure, that we at least unset the property safely at the end.

Typically you run the inspector script in a CI environment with a shared IDEA installation and we do not want to interfere with subsequent runs.